ocw_data_parser/ocw_data_parser_test.py

```diff
@@ -750,7 +750,9 @@ def test_none(field):
 def test_archived_course_fields(ocw_parser):
     """Assert that fields related to handling archived courses are passed through more or less"""
     assert ocw_parser.parsed_json["dspace_handle"] == ""
-    assert ocw_parser.parsed_json["is_update_of"] == "389a811a12a85b2225f41dca56699e0c"
+    assert ocw_parser.parsed_json["is_update_of"] == [
+        "389a811a12a85b2225f41dca56699e0c"
+    ]
```

Comment:
would it be a good idea to also test that an array makes its way through here? just so we have like a regression test

Reply:
I'm not sure I understand, this is asserting an array already. Do you mean an array with more than one item?

Reply:
haha yes that is sort of what I meant - but I think you're right that it's fine as-is :)